ui/src/app/(home)/page.tsx

```diff
@@ -36,7 +37,9 @@ const Home = async () => {
                             src="/uhgroupings/uh-groupings-logo-large.svg"
                             alt="UH Groupings"
                             width={365.5}
-                            height={292.5}
+                            height={293.688}
+                            sizes="(max-width: 574px) 210px, (max-width: 991px) auto, (min-width: 992px) 365.5px"
+                            className="w-[210px] h-[auto] sm:w-[auto] sm:h-[auto] lg:w-[365.5px] lg:h-[293.688px]"
```

Comment:
Are you able to avoid having `width`, `height` and `className` all at the same time? I'm thinking you should be able to remove `className` and then maybe `width` and `height` too.

Reply:
The width and height keep the image's shape and prevent the layout from shifting while the image loads by reserving the right amount of space. The className ensures the image adjusts appropriately across various screen sizes. The size attribute's function is only choosing the best image size for different screens, making the page load faster.

Reply:
The main issue is that having both `width` and `height` with `className` is very confusing. Is there actually no way to get rid of one or the other?

Reply:
Do you mean like the width={337.5} and height={52.5}?

Reply:
Yes